BEMT.m

```diff
@@ -0,0 +1,189 @@
+%
+% Eli-TAARG GitHub link: <https://github.com/TAARG-Education/Eli-TAARG>
+%
+% Eli-TAARG is distributed in the hope that it will be useful,
+% but WITHOUT ANY WARRANTY; without even the implied warranty of
+% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+% General Public License for more details.
+% <http://www.gnu.org/licenses/>.
+%
+% ==============================================================================================
+% |Name        : Function BEMT (1, 2 and 3 Methods)                                           |
+% |Author      : Crispino Umberto & Di Maio Daniele                                           |
+% |              University of Naples Federico II                                             |
+% |Version     : 1.0.0                                                                        |
+% |Date        : 15/12/2020                                                                   |
+% |Modified    : 26/02/2021                                                                   |
+% |Description : Using flight conditions and the geometry of the propeller, through three     | 
+%                different methods, the induced angle of incidence is obtained                |
+% |Reference   : "Lezioni di Aerodinamica dell'ala rotante" by Renato Tognaccini              |
+% |Input       : Geometry of the propeller (number of blades (N), radius (R), number of       |
+%                sections (ns) and radius of the hub (r_hub)) and flight conditions           |
+%                (V and rpm)                                                                  |  
+% |Output      : Induced incidence angle valued by Methods N1, N2 and N3                      |
+% |Note        :                                                                              |
+% ==============================================================================================
+
+function [alpha_i_deg, beta_deg, c, Cl_alpha]=BEMT(V, rpm, R, r_hub, N, ns)
+
+p = 1;
+while p < ns+1
+s=input(['INSERT THE VALUE OF THE CHORD [m] AT THE STATION ',num2str(p), ' : ']);
+c(p)=s;
+p = p+1;
+end
+p = 1;
+while p < ns+1
+f=input(['INSERT THE VALUE OF THE Cl_ALPHA [1/rad] AT THE STATION ',num2str(p), ' : ']);
+Cl_alpha(p)=f;
+p = p+1;
+end
+p = 1;
+while p < ns+1
+d=input(['INSERT THE VALUE OF THE PITCH ANGLE [deg] AT THE STATION ',num2str(p), ' : ']);
+beta_deg(p)=d;
+p = p+1;
+end
+
+omega  = convangvel(rpm, 'rpm', 'rad/s');
+r      = linspace(r_hub, R*0.99, ns);
+beta   = convang(beta_deg, 'deg', 'rad');
+r_ndim = r/R;
+x      = linspace(r_hub,R,100);
+
+lam    = V/(omega*R);
+sigma  = N*c./(pi*R);
+Vt     = omega*R;
+Vr     = sqrt((V^2)+ (omega*R.*r_ndim).^2);
+phi    = atan((V)./(omega*r)); 
+
+disp('Which theory do you want to use?')
+disp('1 - IMPULSIVE THEORY')
+disp('2 - WHIRLING THEORY WITH SMALL DISTURBANCES')
+disp('3 - WHIRLING THEORY')
+
+l= input(['']);

```

Comment:
This one should be an input of the function.

Reply:
We do the same as 'elican'; if it is necessary we change it.